src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -69,7 +69,7 @@ public static ArgumentMultimap tokenize(String argsString, Prefix... prefixes) {
      * {@code argsString} = "e/hi p/900", {@code prefix} = "p/" and
      * {@code fromIndex} = 0, this method returns 5.
      */
-    private static int findPrefixPosition(String argsString, String prefix, int fromIndex) {
+    protected static int findPrefixPosition(String argsString, String prefix, int fromIndex) {
```

Comment:
Not sure if changing this to protected would violate anything. Is it possible to implement the priority in addCommand without changing this access modifier?

Reply:
Implemented using regex and changed the access modifier back to private!